src/actions/vstudio/_vstudio.lua

```diff
@@ -340,7 +340,13 @@
 		-- Then the system libraries, which come undecorated
 		local system = config.getlinks(cfg, "system", "fullpath")
 		for i = 1, #system do
-			table.insert(links, path.appendextension(system[i], ".lib"))
+			-- Add extension if required
+			local link = system[i]
+			if link:match("[^.]+$") == link then
```

Comment:
Actually, I can see it would work for `x.y.lib`, but does it work for `x.y`? I would this code not add `.lib` in that case, but it should, no?